static/_redirects

```diff
@@ -375,6 +379,21 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 /docs/reference/branding/primary-color /docs/reference/branding
 /docs/reference/branding/secondary-color /docs/reference/branding
 
+# TLS
```

Comment:
I think there are still many other redirects to add here, based on https://github.com/pomerium/documentation/commit/e665ce64b11508ae54bf8e8fb33b0596254608e9:

- /docs/reference/routes/tls-client-certificate
- /docs/reference/routes/tls-custom-certificate-authority
- /docs/reference/routes/tls-downstream-client-certificate-authority
- /docs/reference/routes/tls-downstream-server-name
- /docs/reference/routes/tls-skip-verification
- /docs/reference/routes/tls-upstream-allow-renegotiation